db/seeds.rb

```diff
@@ -1,125 +1,212 @@
+    )
+
+    TechhireLocation.create(
+      city: "Minneapolis",
+      state: "Minnesota"
+    )
+
+    TechhireLocation.create(
+      city: "Kansas City",
+      state: "Kansas"
+    )
+
+    TechhireLocation.create(
+      city: "Memphis",
+      state: "Tennessee"
+    )
+
+    TechhireLocation.create(
+      city: "City of Kearney and Buffalo County",
+      state: "Kentucky"
+    )
+
+    TechhireLocation.create(
+      city: "Eastern Kentucky",
+      state: "Kentucky"
+    )
+
+    TechhireLocation.create(
+      city: "Nashville",
+      state: "Tennessee"
+    )
+
+    TechhireLocation.create(
+      city: "Rochester",
+      state: "New York"
+    )
+
+    TechhireLocation.create(
+      city: "Detroit",
+      state: "Michigan"
+    )
+
+    TechhireLocation.create(
+      city: "City of Kearney and Buffalo County",
+      state: "Kentucky"
+    )
+
+    TechhireLocation.create(
+      city: "San Francisco",
+      state: "California"
+    )
+
+    TechhireLocation.create(
+      city: "Albuquerque",
+      state: "New Mexico"
+    )
+
+    TechhireLocation.create(
+      city: "Chattanooga",
+      state: "Tennessee"
+    )
+
+    TechhireLocation.create(
+      city: "Portland",
+      state: "Oregon"
+    )
+
+    TechhireLocation.create(
+      city: "Dover",
+      state: "Delaware"
+    )
+
+    TechhireLocation.create(
+      city: "Anchorage",
+      state: "Alaska"
+    )
+  end
+
+  #Companies
+  def build_uncontacted_companies
+    i = 1
+    10.times do
+      Company.create(
+      name: "user#{i}",
+      organization: "company#{i}",
+      title: "head person#{i}",
+      state: "MO",
+      city: "St. Louis",
+      email: "user#{i}@email.com",
+      hiring: 1,
+      hire_count: 5,
+      status: "uncontacted"
+      )
+      i = i + 1
+    end
+  end
+
+  def build_contacted_companies
+    i = 11
+    10.times do
+      Company.create(
+        name: "user#{i}",
+        organization: "company#{i}",
+        title: "head person#{i}",
+        state: "CO",
+        city: "Denver",
+        email: "user#{i}@email.com",
+        hiring: 1,
+        hire_count: 5,
+        status: "contacted")
+      i = i + 1
+    end
+  end
+
+  def build_confirmed_companies
+    i = 21
+    9.times do
+      Company.create(
+        name: "user#{i}",
+        organization: "company#{i}",
+        title: "head person#{i}",
+        state: "KS",
+        city: "Witchita",
+        email: "user#{i}@email.com",
+        hiring: 1,
+        hire_count: 5,
+        status: "confirmed"
+      )
+      i = i + 1
+    end
+  end
+
+  def build_dead_companies
+    i = 31
+    10.times do
+      Company.create(
+        name: "user#{i}",
+        organization: "company#{i}",
+        title: "head person#{i}",
+        state: "OK",
+        city: "Oklahoma City",
+        email: "user#{i}@email.com",
+        hiring: 1,
+        hire_count: 5,
+        status: "dead"
+      )
+      i = i + 1
+    end
+  end
+end
+
+Seed.new
```

Comment:
Love the updated seed file!

We can do this in another pull request but this could also use an example admin user and super user!